### ISSUE TITLE
Tabs vs. Spaces

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peterhelin/pydelicious#12]
```
Not strictly a problem but an issue to be resolved.

pydelicious.py uses spaces for indentation (mostly, but a quick look just
now shows its a mix... goodness...)

my own preference leans towards tabs, now used in tests/* and tools/*

*Votes for tabs or spaces please?* Or does the project owner which to have
veto right? :)

I vote for using tabs throughout all of the code.


mvrgr, Berend
```

Original issue reported on code.google.com by `berend.v...@gmail.com` on 15 Oct 2007 at 3:49

**Comment by GoogleCodeExporter** [comment]
```
+1 for spaces only. People use different settings for tabs (4 spaces vs 6 
spaces vs 8
spaces, etc) and that can lead to ugly-lloking, especially if there are more 
than 2
people editing a file. See this thread for more :) :
http://mail.python.org/pipermail/python-list/2003-January/183758.html
```

Original comment by `dfdes...@gmail.com` on 15 Oct 2007 at 4:25

**Comment by GoogleCodeExporter** [comment]
```
yeah, I know its a Big Doctrinal Conflict [tm]
hope it doesn't flame up here :)

But even when using spaces, we need to arrive at a consensus on /how many/ 
spaces to
use for indent. This means people need to fix their editor rc's/config for this
project anyway (while I havent seen indents of 2 or 8 spaces, but anyway, I say 
its
1tab==4spaces) Btw, tabs don't change in number, and I, naively, expect most 
editors
allow you configure wether to expand tabs to spaces and how many spaces to use. 
(I
don't have a problem with it anyway).

So, in short... 
- Use tabs
- Use four spaces.

PS: please s/which/wish/ in my first post
```

Original comment by `berend.v...@gmail.com` on 15 Oct 2007 at 4:50

**Comment by GoogleCodeExporter** [comment]
```
I'm voting for 4 spaces :-)
```

Original comment by `gregpin...@gmail.com` on 15 Oct 2007 at 5:34

**Comment by GoogleCodeExporter** [comment]
```
:(

OK.
```

Original comment by `berend.v...@gmail.com` on 15 Oct 2007 at 6:04

**Comment by GoogleCodeExporter** [comment]
```
I'm voting for 4 spaces too :)
```

Original comment by `alexco...@gmail.com` on 16 Oct 2007 at 8:44

**Comment by GoogleCodeExporter** [comment]
```
Changed and committed.
```

Original comment by `berend.v...@gmail.com` on 7 Nov 2007 at 9:16
- Changed state: **Fixed**
